static/js/popovers.js

```diff
@@ -42,14 +42,15 @@ function load_medium_avatar(user_email) {
 }
 
 function user_last_seen_time_status(user_id) {
-    if (presence.get_status(user_id) === "active") {
+    var status = presence.get_status(user_id);
+    if (status === "active") {
         return i18n.t("Active now");
     }
-
-    if (people.get_person_from_user_id(user_id).is_bot) {
```

Comment:
Yeah. Some bots like `welcome-bot` and `new user bot` won't get inside this if statement if they are in realms other than Zulip. So checking for `is_bot` is not the best approach right now until we fix that. The change also handles users who have not logged in for a long time and have no presence info.